test-app/.eslintrc.js

```diff
@@ -53,7 +53,9 @@ module.exports = {
         'plugin:@typescript-eslint/eslint-recommended',
         'plugin:@typescript-eslint/recommended',
       ],
-      rules: {},
+      rules: {
+        'no-undef': 'off',
```

Comment:
Why do we need to disable this rule?

Reply:
There are some global DOM types that ESLint does not recognize. Specifically `(animation as CSSTransition)` in `bs-modal-test.gts`. The docs from typescript-eslint mention that it is recommended to disabled `no-undef` in Typescript files, since Typescript checks for it anyway.

https://typescript-eslint.io/troubleshooting/faqs/eslint/#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors